internal/service/docdb/cluster_instance.go

```diff
@@ -311,11 +328,17 @@ func resourceClusterInstanceRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("availability_zone", db.AvailabilityZone)
 	d.Set("cluster_identifier", db.DBClusterIdentifier)
 	d.Set("dbi_resource_id", db.DbiResourceId)
+	//Commented the below code as aws api does not expose these values and should be uncommented
+	//as soon as the vlues are available in the DescribeDBClusters output
+	//d.Set("enable_performance_insights", db.EnablePerformanceInsights)
```

Comment:
Won't it cause an infinite change of these two attributes during plan and apply? You cannot update these fields in Reading function, so how will the state get these values then?

Reply:
Only when the API gets updated, we can update the read finction with the new values.